docker-compose.yml

```diff
@@ -14,6 +14,12 @@ services:
       - apps/web/.env
     command: pnpm run dev
 
+  elastic:
+    image: elasticsearch:8.14.1
+    environment:
+      - 'discovery.type=single-node'
+      - 'ELASTIC_USERNAME=latitude'
+      - 'ELASTIC_PASSWORD=papapa55'
```

Comment:
Use `secret` to standaraize or change the other to `papapa55`

Reply:
done here https://github.com/latitude-dev/latitude-llm/pull/7